tube-player/README.md

```diff
@@ -37,29 +40,54 @@ This workshop has been set up to provide you with optional content to allow you
 
 The workshop utilizes the latest Uno Platform features, including:
 
-- [C# Markup](xref:Reference.Markup.GettingStarted)
-- [Extensions](xref:Overview.Features)
-  - [MVUX](xref:Overview.Mvux.Overview)
-    - [Pagination](xref:Overview.Mvux.Advanced.Pagination)
-    - [FeedView](xref:Overview.Mvux.FeedView)
-      - NoneTemplate
-      - ErrorTemplate
-    - [Refit](xref:Overview.Http)
-    - [Region navigation](xref:Overview.Navigation)
-    - [Serialization](xref:Overview.Serialization)
+### Environment setup & Inner loop
+
+- [Hot Reload](xref:Uno.Features.HotReload)
+- [Uno.Check](xref:UnoCheck.UsingUnoCheck)
+
+### Cross-platform development
+
+- [Uno.Resizetizer](xref:Uno.Resizetizer.GettingStarted)
+  - [UnoIcon](xref:Uno.Resizetizer.GettingStarted#unoicon)
+  - [UnoSplashScreen](xref:Uno.Resizetizer.GettingStarted#unosplashscreen)
+- [Template Wizard](xref:Uno.GettingStarted.UsingWizard)
+
+### Battle-tested features for apps
+- [Extensions](xref:Uno.Extensions.Overview)
+    - [Hosting environments](xref:Uno.Extensions.Hosting.Overview#hosting-environments)
+    - [Navigation with regions](xref:Uno.Extensions.Navigation.Advanced.Panel)
+    - [Refit endpoints](xref:Uno.Extensions.Http.HowToRefit)
+    - [Serialization](xref:Uno.Extensions.Serialization.Overview)
+- [C# Markup](xref:Uno.Extensions.Markup.Overview)
```

Comment:
```suggestion
- [Extensions](xref:Uno.Extensions.Overview)
    - [Hosting environments](xref:Uno.Extensions.Hosting.Overview#hosting-environments)
    - [Navigation with regions](xref:Uno.Extensions.Navigation.Advanced.Panel)
    - [Refit endpoints](xref:Uno.Extensions.Http.HowToRefit)
    - [Serialization](xref:Uno.Extensions.Serialization.Overview)
    - [C# Markup](xref:Uno.Extensions.Markup.Overview)
```

@lukeblevins any reason why C# Markup is not under Extensions like the others ? 
Since C# Markup is from Uno.Extensions as well